keyboards/idobao/montex/v2/config.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 peepeetee (@peepeetee)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x6964  // "id"
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0002
```

Comment:
VIA issues may require changing the `PRODUCT_ID` too (although the matrix layout seems to be exactly the same as on V1, so the basic VIA JSON for V1 will still work for V2, changing from `RGBLIGHT_ENABLE` to `RGB_MATRIX_ENABLE` will require a separate VIA JSON if you want to have proper effect names there; also the backlight feature was in V1, but not in V2).

Reply:
product ID changed to `0x0127` + new VIA json pending at the-via